nix/default.nix

```diff
@@ -27,7 +37,13 @@ let
   # attribute paths when imported as flake/non-flake.
   haskellNixConfig = if isInFlake then haskellNix.internal.config else haskellNix.config;
 
-  extraOverlays = haskellNixOverlays ++ ownOverlays;
```

Comment:
It's important that we do it this way, don't revert this! just add the iohk-nix bit back in